lib/minesweeper_2pl/boardcli.rb

```diff
@@ -0,0 +1,57 @@
+module Minesweeper_2pl
+  class BoardCli
+
+    attr_accessor :message, :board, :show_bombs
+
+    NEWLINE = "\n"
+    ALPHA_LEFT = "  "
+    ALPHA_RIGHT = " "
+    CELL_LEFT = "|  "
+    CELL_RIGHT = "  "
+    CELL_END = "  |"
+    CELL_DIVIDER = "+======"
+    HEADER_CELL_LEFT = "   "
+    HEADER_CELL_RIGHT = "   "
+    INTRO = "    "
+
+    def print_message(string)
+      puts string
+    end
+
+    def set_board(positions, rowsize)
```

Comment:
1) `positions` and `rowsize` are both attributes of `Board`, aren't they? Could you design this method to take a `Board` object as a parameter instead?
2) It looks to me like this method is building up a string representation of a board. Is `set_board` a descriptive name for that behavior or is there a better name for this?
3) The structure of this method looks like it has a few parts: reading the contents of a cell and building up an representation based on its state, connecting the cells into rows, connecting the rows into a full board representation. Could you make this more readable by extracting some of those sub-responsibilities to their own named methods? Ideally, the body of method would read more like English than code.